app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/components/DatastoresStepForm/helpers.js

```diff
@@ -22,34 +20,16 @@ export const targetDatastoreAvailableSpace = (targetDatastore, datastoresStepMap
 };
 
 export const sourceDatastoreInfo = sourceDatastore =>
-  sprintf(
-    __('%s \\ %s (%s)'),
-    sourceDatastore.providerName,
-    sourceDatastore.name,
-    numeral(datastoreUsedSpace(sourceDatastore)).format('0.00b')
-  );
+  sprintf(__('%s \\ %s'), sourceDatastore.providerName, sourceDatastore.name);
 
 export const targetDatastoreInfo = (targetDatastore, datastoresStepMappings) =>
-  sprintf(
-    __('%s \\ %s (%s avail)'),
-    targetDatastore.providerName,
-    targetDatastore.name,
-    numeral(targetDatastoreAvailableSpace(targetDatastore, datastoresStepMappings)).format('0.00b')
-  );
+  sprintf(__('%s \\ %s'), targetDatastore.providerName, targetDatastore.name);
 
-export const targetDatastoreTreeViewInfo = (targetDatastore, sourceDatastores) => {
-  const { total_space, free_space } = targetDatastore;
-  const availableSpace = free_space - totalUsedSpace(sourceDatastores);
-
-  return sprintf(
-    __('%s (%s total, %s avail)'),
-    targetDatastore.name,
-    numeral(total_space).format('0.00b'),
-    numeral(availableSpace).format('0.00b')
-  );
-};
+export const targetDatastoreTreeViewInfo = targetDatastore => targetDatastore.name;
 
-export const errorMessage = __('The size of the selected source datastores exceeds the available space in the target datastore'); // prettier-ignore
+export const errorMessage = __(
```

Comment:
waitttt i think we still want this to all be on one line... the `__(` n stuff

Reply:
Good catch - thanks!